src/aliceVision/sfm/pipeline/sequential/ReconstructionEngine_sequentialSfM.cpp

```diff
@@ -1666,6 +1666,44 @@ void ReconstructionEngine_sequentialSfM::getTracksToTriangulate(const std::set<I
   }
 }
 
+namespace {
+
+struct ObservationParams
+{
+  const View* view = nullptr;
```

Comment:
is it necessary? It is never used in the code.
In case wouldn't be better to avoid the raw pointer?

Reply:
Fixed now.